tests/index.js

```diff
@@ -1,120 +1,162 @@
+#!/usr/bin/env node
```

Comment:
can't we use async/await in those tests? IMO it'd make things more readable.

Reply:
It is possible. The tests right now are nothing but a move of the tests of master to this branch.